### ISSUE TITLE
Assign graphics to new foods

### ISSUE THREAD

**Issue by angband-import** [angband/angband#3801]
**Reported by takkaria on 24 Apr 2013 11:17 UTC**
Hard Biscuit & al don't have them at present.  Might have to ask Shockbolt to make new ones?

**Comment by angband-import** [comment]
**Comment by draconisPW on 25 Apr 2013 11:10 UTC**
Hard Biscuit and Fine Wine already have their tile, since they were present before. Flask of Whisky could use the tile from the old Fine Ale.

That's how I remapped them myself:

- graf-dvg and graf-shb

```
K:food:Hard Biscuit:0x82:0x82
K:food:Flask of Whisky:0x82:0x80
K:food:Pint of Fine Wine:0x82:0x81
```

- graf-new

```
K:food:Hard Biscuit:0x8E:0x82
K:food:Flask of Whisky:0x8E:0x80
K:food:Pint of Fine Wine:0x8E:0x81
```

- graf-nmd

```
K:food:Hard Biscuit:0x89:0x88
K:food:Flask of Whisky:0x89:0x83
K:food:Pint of Fine Wine:0x89:0x83
```

- graf-xxx

```
K:food:Hard Biscuit:0x8B:0x82
K:food:Flask of Whisky:0x8A:0x95
K:food:Pint of Fine Wine:0x8A:0x96
```

**Comment by angband-import** [comment]
**Comment by takkaria on 25 Apr 2013 12:51 UTC**
Thanks for those, fixed in master rev 9dcc50f.
